Modules/Relationships/src/Relationships.Application/Identities/IdentityDeleter.cs

```diff
@@ -20,11 +21,12 @@ public IdentityDeleter(IMediator mediator, IDeletionProcessLogger deletionProces
 
     public async Task Delete(IdentityAddress identityAddress)
     {
-        await _mediator.Send(new DeleteRelationshipsOfIdentityCommand(identityAddress));
+        await _mediator.Send(new DecomposeRelationshipsOfIdentityCommand(identityAddress));
```

Comment:
We should have a single command called `DecomposeAndAnonymizeRelationshipsOfIdentityCommand`. The reason for this is that we want this to happen in an atomic step, in order to not confuse the runtime.

Reply:
If we combine it into one, I think we could revert the cache clearing since we aren't querying the DB twice anymore

Reply:
Hmm, true. Good catch. Do it. :)